lib/renderer.js

```diff
@@ -1164,10 +1183,12 @@
         // 4. The "include-ancestor-masks" config option is NOT set
         // 5. None of the layers has zero bounds. Sometimes they aren't computed and set to all 0's
         // 6. The layer is clipped, in which case layer.bounds is the clipped size and not the layer size
-        var hasComplexTransform = (component.hasOwnProperty("scale") && component.scale % 1 !== 0) ||
-                component.hasOwnProperty("width") || component.hasOwnProperty("height"),
+        var layer = component.layer,
+            hasComplexTransform = layer &&
```

Comment:
I'll be honest @chrisbank, I don't really understand your response.  But I created a PR with my very limited fix, so if you have a moment to check that out, please do.  https://github.com/adobe-photoshop/generator-assets/pull/393